### DISCUSSION TITLE
BC0426C6-2546-500A-B0FC-C9AE8ACB3204

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#217] [General]
# TRAPPIST-1 (Legacy)

This addon adds fictional textures for the planets orbiting the ultracool dwarf star TRAPPIST-1 to Celestia. All seven planets are similar in size and mass to Earth, and three or four orbit within the habitable zone. The system is a target for atmospheric spectroscopy by the James Webb Space Telescope. The textures in this version of the addon are from the 2018 NASA artist's impression, and can be set to 2K, 4K, or 8K resolution depending on your device's capabilities.

https://celestia.mobi/addon?item=BC0426C6-2546-500A-B0FC-C9AE8ACB3204

**Comment by PhilThePlanetball** [comment]
this addon getting an update?

**Comment by foggy-from-trappist-1f** [comment]
Why are the planets' textures so blurry? Or am I the only one with this problem?
